src/AvTranscoder/transcoder/StreamTranscoder.cpp

```diff
@@ -569,7 +571,26 @@ bool StreamTranscoder::processTranscode()
 
     // Transform
     CodedData data;
-    if(decodingStatus)
+    bool continueProcess = true;
+    for(size_t index = 0; index < decodingStatus.size(); ++index)
+    {
+        if(!decodingStatus.at(index))
+        {
+            if(!_filterGraph->hasFilters() || !_filterGraph->hasBufferedFrames(index))
+            {
+                continueProcess = false;
+                continue;
```

Comment:
Did you consider to user __break__ instead of __continue__? Because in this case, do we need to continue to get the remaining buffers in the filter graph?

Reply:
Sure, it is better! See 0e49751